README.md

```diff
@@ -27,6 +27,7 @@ This project is still under development. The following features with the check m
 
 - CMake >= 3.8.0
 - Rust >= 1.19.0
+- binutils >= 2.22
```

Comment:
I don't think it's necessary. And it's usually a dependency of gcc.

Reply:
How about requiring a newer version of gcc?
Actually I don't know the corresponding gcc version.

Reply:
Sounds more reasonable to me. Whatever works for grpc/grpc.

Reply:
According to https://grpc.io/about/

gcc **4.4, 4.6**, 5.3 are supported while in #274 people are using gcc 4.8 and 4.9.
It's like the version of binutils is independent to gcc.